src/main/webapp/userdir.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Student Recommendations</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles/fonts.css">
+    <link rel="stylesheet" href="/styles/common.css">
+    <link rel="stylesheet" href="/styles/userdir.css">
+  </head>
+  <body>
+    <div class="header">
+      <h1>StudentRecs</h1>
+      <h3>a recommendation system for students, by students!</h3>
+
```

Comment:
nit: is this whitespace necessary? you haven't used it anywhere else in the document so maybe looks a little odd.